src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,31 +1,41 @@
+import { Card, WithStyles } from '@material-ui/core';
+import CardMedia from '@material-ui/core/CardMedia';
+import { withStyles } from '@material-ui/core/styles';
 import * as React from 'react';
 import { RouteComponentProps } from 'react-router-dom';
 import { Movie } from '../App/App';
-import './MovieCard.scss';
+import movieCardStyles from './MovieCard.styles';
 
 interface Props extends RouteComponentProps<any> {
     handleMovieCardClick: (title: string) => void;
     movie: Movie;
 }
 
-export default React.memo(({ handleMovieCardClick, movie, history }: Props) => {
+type PropsWithStyles = Props & WithStyles<'card' | 'media' | 'titleDate' | 'title' | 'date' | 'genres'>;
```

Comment:
Not recommend to type properties as string literals not to miss them during refactoring, however I get the case.

Reply:
This is recommended way to type classes prop.

Reply:
I don't like this way, but it is recommended way to type classes prop.

Reply:
Yep, I got the case, I'm okay with that.